src/plugin/Component.vue

```diff
@@ -10,6 +10,7 @@
       :type="cellsInputTypes[index]"
       @focus="focusedCellIdx = index"
       @keydown.delete="onCellErase(index, $event)"
+      @keydown="onKeyDown($event)"
```

Comment:
You don't actually need to explicitly pass the "$event" parameter - it is automatically passed by default. You need to specify it explicitly only in case when  you want to pass a parameter after it.